apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Update year also.

Reply:
Since this is an old file, the year won't be updated.